lodepng.cpp

```diff
@@ -5871,11 +5871,15 @@ unsigned lodepng_encode(unsigned char** out, size_t* outsize,
     if(!state->error) {
       state->error = lodepng_convert(converted, image, &info.color, &state->info_raw, w, h);
     }
-    if(!state->error) preProcessScanlines(&data, &datasize, converted, w, h, &info, &state->encoder);
+    if(!state->error) {
+      state->error = preProcessS1canlines(&data, &datasize, converted, w, h, &info, &state->encoder);
```

Comment:
Thanks for fixing! One more thing: the function name has a typo: preProcessS1canlines instead of preProcessScanlines